<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1380</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_CameraView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>500</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Camera View</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_MotionControl">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>700</width>
      <height>320</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Motion Control</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_LogWindow">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>640</width>
      <height>500</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Log Window</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Configuration">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>500</y>
      <width>340</width>
      <height>320</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Configuration</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_SystemStatus">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>500</y>
      <width>280</width>
      <height>320</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>System Status</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1380</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
